src/gui/pedalboard/editors/Port.h

```diff
@@ -2,21 +2,30 @@
 
 #include <pch.h>
 
+enum PortType
+{
+    audio = 0,
+    modulation,
+};
```

Comment:
I wonder if we should make this an `enum class`? Otherwise, every bit of code that includes this header will have a naming conflict with `audio` and `modulation`.

Reply:
Yeah good call, I think that could save a weird error later on